tox.ini

```diff
@@ -85,7 +85,7 @@ allowlist_externals = /usr/bin/env
 description = Scenario tests
 deps =
     pytest
-    ops-scenario>=4.0.4
+    ops-scenario<5
```

Comment:
@PietroPasotti I tried `>=5`, unpinned ops and ran `tox -re`, but there were `_fs` errors and others.